frontend/src/pages/TriviaWelcome.js

```diff
@@ -1,11 +1,36 @@
-//import { Transition } from "react-spring/renderprops";
+import { useSpring, animated } from "react-spring";
 import styled from "styled-components/macro";
+import { NextButtonStyle } from "../components/NextButtonStyle";
+import { BsArrowRightShort } from "react-icons/bs";
+import { Link, useHistory } from "react-router-dom";
+import logo from "../images/whiteMan.png";
+import React from "react";
 
 export default function TriviaWelcome() {
+  const history = useHistory();
+  const handleSubmit = (event) => {
+    history.push("/questions");
+  };
+
```

Comment:
You can do this directly in the calling component, no need to define it here and pass it down. Also, the name indicates this handles a submit (form event).. if so, call prevent the default behavior, if not please rename